.github/workflows/sub-find-cached-disks.yml

```diff
@@ -65,35 +65,33 @@ jobs:
       # - lwd_tip_disk
       # - zebra_tip_disk
       # - zebra_checkpoint_disk
+      # TODO evaluate if ${{ toJSON(false) }} can be used inside an external script to move this logic out of the workflow
```

Comment:
I think toJSON doesn't actually do anything to a boolean value?
https://quickref.me/json

Let's try printing the value it produces, and then just using that directly?

Reply:
I'll look into this